packages/cli/tsconfig.json

```diff
@@ -1,20 +1,16 @@
 {
   "compilerOptions": {
-    "strict": true,
-    "noEmitOnError": true,
-    "noFallthroughCasesInSwitch": true,
-    "moduleResolution": "node",
-    "module": "commonjs",
-    "target": "ES2021",
-    "esModuleInterop": true,
     "allowJs": true,
-    "lib": ["ES2021"],
-    "resolveJsonModule": true,
-    "sourceMap": false,
+    "noImplicitReturns": false,
+    "noUnusedLocals": false,
+    "noUnusedParameters": false,
```

Comment:
So these are set to `false` because our `cli` code isn't compatible with the strictness of `@vercel/style-guide/typescript`, right? 

We should track this in Linear as a follow up to improve our code quality to match the strictness of the style-guide. Sorta like API

Reply:
Yep.